src/model/constants/index.ts

```diff
@@ -43,6 +43,7 @@ export const Constants = {
   AWS_HA_MASTERS_MAX_NODES: 3,
 
   AZURE_CAPZ_VERSION: '20.0.0',
+  AZURE_NON_EXP_MACHINE_POOLS_VERSION: '17.1.0',
```

Comment:
This is the minimum version required for supporting `MachinePools` and `AzureMachinePools` from the non-experimental API groups. For testing purposes, this may need to be temporarily changed